plugins/browser-plugin-form-tracking/src/helpers.ts

```diff
@@ -70,7 +70,7 @@ const defaultFormTrackingEvents = [
 ];
 
 export interface FormTrackingOptions {
-  forms?: FilterCriterion<HTMLElement>;
+  forms?: FilterCriterion<HTMLElement> | HTMLCollectionOf<HTMLElement>;
```

Comment:
I think this could actually be:

```suggestion
  forms?: FilterCriterion<HTMLElement> | HTMLCollectionOf<HTMLFormElement> | NodeListOf<HTMLFormElement>;
```

You could get a node list with:

```js
let nodeList = document.querySelectorAll('.form')
```

where `.form` is a CSS class to find the forms.

Reply:
Makes sense, added the type.